test/Microsoft.ML.Sweeper.Tests/TestSweeper.cs

```diff
@@ -0,0 +1,756 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.CommandLine;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.EntryPoints;
+using Microsoft.ML.Runtime.RunTests;
+using Microsoft.ML.Runtime.Sweeper;
+using Microsoft.ML.Runtime.Sweeper.Algorithms;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.ML.Sweeper.RunTests
+{
+    using Float = System.Single;
+
+    public sealed partial class TestSweeper : BaseTestBaseline
```

Comment:
>BaseTestBaseline [](start = 46, length = 16)

do you use any functionality from it, or it can just be BaseTestClass?

Reply:
I am able to enable TestSmacSweeper() test which uses RunMtaThread function from base class

Reply:
I find this's actually quite interesting. If you look into RunMTAThread routine it has following code:
```
#if !CORECLR // CoreCLR does not support apartment state settings for threads.
            t.SetApartmentState(ApartmentState.MTA);
#endif
``` 
Which was true for long period of time until you fix it in this PR https://github.com/dotnet/coreclr/pull/15652

This is unrelated to your changes, and I don't want you to do anything about this (I'll create issue to enable MTA) just, as I said earlier, I find this interesting :)